templates/inspec/doc_template.md.erb

```diff
@@ -50,3 +50,13 @@ with `where` as a block or a method.
 <% end -%>
 
 <% end # if plural -%>
+
```

Comment:
You probably want this newline inside of the unless statement. If it isn't, any docs without api links will have an extra newline at the end.